From f87f3fad2b21dee34e38fe15738ca8a2ba46999f Mon Sep 17 00:00:00 2001
From: Amelia Carter <ameliamarycarter@gmail.com>
Date: Wed, 13 Mar 2019 21:19:39 -0700
Subject: [PATCH] Remove that square I hate

---
 dwm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dwm.c b/dwm.c
index 4465af1..8798228 100644
--- a/dwm.c
+++ b/dwm.c
@@ -718,10 +718,10 @@ drawbar(Monitor *m)
 		w = TEXTW(tags[i]);
 		drw_setscheme(drw, scheme[m->tagset[m->seltags] & 1 << i ? SchemeSel : SchemeNorm]);
 		drw_text(drw, x, 0, w, bh, lrpad / 2, tags[i], urg & 1 << i);
-		if (occ & 1 << i)
+		/* if (occ & 1 << i)
 			drw_rect(drw, x + boxs, boxs, boxw, boxw,
 				m == selmon && selmon->sel && selmon->sel->tags & 1 << i,
-				urg & 1 << i);
+				urg & 1 << i); */
 		x += w;
 	}
 	w = blw = TEXTW(m->ltsymbol);
-- 
2.21.0

